script.js

```diff
@@ -3,50 +3,63 @@ let books = [];
 const addBtn = document.getElementById('addBook');
 const listBox = document.getElementById('list');
 
-function displaybooks() {
-  const book = localStorage.getItem('book');
-  if (book === null) {
-    books = [];
-  } else {
-    books = JSON.parse(book);
+/* eslint class-methods-use-this: ["error", { "exceptMethods": ["displayBooks"] }] */
```

Comment:
- [ ] It would be wise to remove disable linters as linter configurations help us to write clean industry standard code.